tools/utils/utils.js

```diff
@@ -25,4 +26,18 @@ const getSheet = async (url) => {
   return sheet;
 };
 
-export { getImsToken, getSheet };
+const getCustomConfig = async (path) => {
+  /* c8 ignore next 3 */
+  if (CONFIGS[path] !== undefined) {
+    return CONFIGS[path];
+  }
+  let config = null;
+  const resp = await fetch(path);
+  if (resp.ok) {
+    config = await resp.json();
+  }
+  CONFIGS[path] = config;
+  return CONFIGS[path];
```

Comment:
```suggestion
  if (CONFIGS[path] !== undefined) {
    return CONFIGS[path];
  }
  const resp = await fetch(path);
  const config = resp.ok ? await resp.json() : null;
  CONFIGS[path] = config;
  return config;
```
nit: slightly smaller way to write this fwiw